app/wallet/bridge/components/ForceInclusionProgress.tsx

```diff
@@ -0,0 +1,301 @@
+              <HStack>
+                {status === "complete" ? (
+                  <CheckCircleIcon color="green.300" boxSize={5} />
+                ) : status === "error" ? (
+                  <Box color="red.300" fontSize="xl">
+                    ⚠️
+                  </Box>
+                ) : (
+                  <SpinnerIcon className="spinner" boxSize={5} />
+                )}
+                <Text fontWeight="bold" fontSize="lg">
+                  {status === "error"
+                    ? "Error"
+                    : status === "complete"
+                    ? "Transaction Complete"
+                    : steps.find((s) => s.key === status)?.label ||
+                      "Processing..."}
+                </Text>
+              </HStack>
+            </Box>
+
+            {/* Error Message */}
+            {error && (
+              <Box
+                p={3}
+                bg="red.900"
+                borderRadius="md"
+                borderWidth={1}
+                borderColor="red.500"
+              >
+                <Text fontSize="sm" color="red.200">
+                  {error}
+                </Text>
+              </Box>
+            )}
+
+            {/* Transaction Hashes */}
+            <VStack spacing={3} align="stretch">
+              {l1Hash && (
+                <Box p={3} bg="whiteAlpha.100" borderRadius="md">
+                  <Text fontSize="sm" fontWeight="bold" mb={1}>
+                    L1 Transaction ({l1Chain?.name || `Chain ${l1ChainId}`}):
+                  </Text>
+                  <Link
+                    href={getExplorerUrl(l1ChainId, l1Hash) || "#"}
+                    isExternal
+                    color="blue.300"
+                    fontSize="sm"
+                    wordBreak="break-all"
+                  >
+                    {l1Hash}
+                  </Link>
+                </Box>
+              )}
+
+              {l2Hash && (
+                <Box p={3} bg="whiteAlpha.100" borderRadius="md">
+                  <Text fontSize="sm" fontWeight="bold" mb={1}>
+                    L2 Transaction ({l2Chain?.name || `Chain ${l2ChainId}`}):
+                  </Text>
+                  <Link
+                    href={getExplorerUrl(l2ChainId, l2Hash) || "#"}
+                    isExternal
+                    color="blue.300"
+                    fontSize="sm"
+                    wordBreak="break-all"
+                  >
+                    {l2Hash}
+                  </Link>
+                </Box>
+              )}
+            </VStack>
+
+            {/* Elapsed Time */}
+            {elapsedTime !== undefined && status === "waiting-l2" && (
+              <Box textAlign="center">
+                <Text fontSize="sm" color="whiteAlpha.700">
+                  Elapsed time: {formatTime(elapsedTime)}
+                </Text>
+                <Text fontSize="xs" color="whiteAlpha.500" mt={1}>
+                  L2 confirmation can take up to 10 minutes
```

Comment:
this is true for optimism and base, but seems like the OP stack specification allows up to 6 hours lol

Reply:
Isn't it up to 12 hours if the sequencer is down? https://docs.optimism.io/stack/transactions/forced-transaction

Reply:
That said, if the point is to exit positions while the sequencer is down, the goal is to get into the inbox queue as early as possible -- the actual latency to confirmation is less important since you can't do much about it.

Though external oracle-based pricing will still wreck people if it's the 12+ hour scenario.